com.codeaffine.eclipse.swt.test/src/com/codeaffine/eclipse/swt/util/UnsafeTest.java

```diff
@@ -57,22 +56,12 @@ public void defineProtectedClass() {
   public void defineClassTwice() {
     byte[] bytes = resourceLoader.load( LOCATION );
 
-    Class<?> first = unsafe.defineClass( CLASS_NAME, bytes, 0, bytes.length, getLoader(), getDomain() );
-    Class<?> second = unsafe.defineClass( CLASS_NAME, bytes, 0, bytes.length, getLoader(), getDomain() );
+    Class<?> first = unsafe.defineClass( CLASS_NAME, bytes, 0, bytes.length, getLoader(), getDomain(), getClass() );
+    Class<?> second = unsafe.defineClass( CLASS_NAME, bytes, 0, bytes.length, getLoader(), getDomain(), getClass() );
 
     assertThat( first ).isSameAs( second );
   }
 
-  @Test
-  public void defineClassThatDoesNotExist() {
-    byte[] bytes = resourceLoader.load( LOCATION );
-
-    Throwable actual = thrownBy(
-      () ->  unsafe.defineClass( "unknown", bytes, 0, bytes.length, getLoader(), getDomain() ) );
```

Comment:
The new `Lookup.defineClass` does not use the provided class name. As long as the bytes represent a valid Java class, it will happily define it. Consequently this test will not pass in JDK9+. It kind of tests internal behaviour of the old `Unsafe.defineClass` method, so I decided to remove it entirely.